client/my-sites/media-library/filter-bar.jsx

```diff
@@ -97,23 +99,44 @@ export class MediaLibraryFilterBar extends Component {
 		);
 	}
 
+	renderSearchSection() {
+		if ( this.props.filterRequiresUpgrade ) {
+			return null;
+		}
+
+		return (
+			<Search
+				analyticsGroup="Media"
+				pinned
+				fitsContainer
+				onSearch={ this.props.onSearch }
+				initialValue={ this.props.search }
+				placeholder={ this.getSearchPlaceholderText() }
+				delaySearch={ true } />
+		);
+	}
+
+	renderPlanStorage() {
+		const eventName = 'calypso_upgrade_nudge_impression';
+		const eventProperties = { cta_name: 'plan-media-storage' };
+		return (
+			<PlanStorage siteId={ this.props.site.ID }>
+				<TrackComponentView eventName={ eventName } eventProperties={ eventProperties } />
```

Comment:
fun fact! when I'm just passing named properties like this I find the spread syntax helpful…

```js
<TrackComponentView { ...{
	eventName,
	eventProperties
} } />
```